cephalopod/cli.py

```diff
@@ -0,0 +1,63 @@
+import click
+
+from celery.bin.celery import CeleryCommand, command_classes
+
+from . import crawler
+from .app import app
```

Comment:
Do we really want to go for an importable app object? I think that will actually result in the dev server not surviving syntax errors.

See https://github.com/indico/indico/blob/master/indico/cli/core.py#L41-L46 (with the normal `FlaskGroup` instead, and `create_app=_create_app` like [here](https://github.com/indico/indico/blob/master/indico/cli/util.py#L34-L36))